docs/admin/setup/distributed.rst

```diff
@@ -533,3 +537,163 @@ different ports and a unique instance ID for each instance.
+
+#. Start the docker containers by running the following command:
+
+    .. code:: bash
+
+        docker compose -f docker/test-server-multi-node-mysql-localci.yml up
+
+#. You can now access artemis on ``http(s)://localhost`` and the registry on ``http://localhost:8761``.
+
+
+Separate Core and BuildAgent Instances
+""""""""""""""""""""""""""""""""""""""
+
+In a multi instance setup, it is possible to separate the core and build agent instances. This can be useful if you want
+to have a dedicated instances for running the build agents. The core instances will then only be responsible for handling
+the core functionality of Artemis (e.g. managing courses, exercises, users, etc.). The build agent instances will only be
+responsible for running the build agents. This can be useful if you want to scale the build agents independently from the
+core instances. The setup will be similar to the one described in :ref:`Running multiple instances locally with Docker`.
+The main differences are:
+
+* The artemis instances will be separated into core and build agent instances:
+
+    * artemis-app-node-1: using following spring profile: ``prod,localvc,localci,core,scheduling,docker``
+    * artemis-app-node-2: using following spring profile: ``prod,buildagent``
+    * artemis-app-node-3: using following spring profile: ``prod,localvc,localci,core,docker``
+
+* The load balancer will not forward requests to the build agent instance (node-2).
+
+
+#. Follow the steps described in :ref:`Running multiple instances locally with Docker`. However, do not run the
+   ``docker compose`` command to start the docker containers yet.
+
+#. Add the following environment variables to the file ``docker/.env``:
+
+    .. code:: bash
+
+        ARTEMIS_NODE_1_ENV_FILE=./artemis/config/node1-core.env
+        ARTEMIS_NODE_2_ENV_FILE=./artemis/config/node2-buildagent.env
+        ARTEMIS_NODE_3_ENV_FILE=./artemis/config/node3-core.env
+
+#. Comment out the following line in the file ``docker/nginx/artemis-upstream-multi-node.conf``:
+
+    .. code:: bash
+
+        server artemis-app-node-1:8080;
+        # server artemis-app-node-2:8080;
+        server artemis-app-node-3:8080;
+
+#. Start the docker containers by running the following command:
+
+    .. code:: bash
+
+        docker compose -f docker/test-server-multi-node-mysql-localci.yml up
+
+    .. hint::
+
+        You may run into the following error when starting the containers
+        ``No member group is available to assign partition ownership...``. To fix this, you can first start the following
+        containers:
+        ``docker compose -f docker/test-server-multi-node-mysql-localci.yml up mysql jhipster-registry activemq-broker artemis-app-node-1``.
+        After these containers are up and running, you can start the remaining containers:
+        ``docker compose -f docker/test-server-multi-node-mysql-localci.yml up artemis-app-node-2 artemis-app-node-3 nginx``.
```

Comment:
FYI: This will be somewhat fixed by #8473. While these errors may still occur at the beginning, after a couple minutes it will fix itself.

Reply:
I mentioned that it should fix itself after a few seconds. But kept the hint just in case it doesn't